Table of downloads as linechart / history of subscriptions

poschi3 opened the pull request:
I've made a prototype to chart the table of downloads. If you say that all your subscribers are downloading your episodes automatically within the first 24/48 hours, then you can interpret the chart as history of number of subscribers.

Here is an example (with removed numbers):
![grafik](https://user-images.githubusercontent.com/825911/76165571-65bb2000-6158-11ea-81ed-b756b34bb9f2.png)

There are a lot of bugs, missing things and shortcuts. I'm posting this to get some feedback and to check that I don't make something complete wrong.

eteubert commented:
> check that I don't make something complete wrong.

Makes sense on the first glance :) Some lose thoughts:

- not sure the "total" makes sense in this view, I feel it's rather distracting 
- I'd be careful to label this as "subscribers" but your general thought is correct -- however the main data you can see from this is the relative growth
- maybe hide points without values (not sure if this is hard to do with the chart framework but the green line for example should just stop one point earlier)

noqqe commented:
Whats the status on this? Would love to see this merged <3

poschi3 commented:
@noqqe I need some time to fix a lot of problems :-)

poschi3 commented:
![image](https://user-images.githubusercontent.com/825911/91670526-c4b58c80-eb1e-11ea-9c6f-92b124e84589.png)

@eteubert Thank you for your input. I had some time and was motivated by the new Podlovers Podcast and fixed some things:
* Rebased to beta branch
* Points with no values: The null-value handling is now fixed and the chart is configured to ignore null values
* The x axis was ordered by the titles of the episodes, but when there is a episode with something that doesn't begins with "EB00x" the chart was messed up. Now it's in the order of the (csv-)data. Unfortunately we lost any title in the chart. I was not able to fix it. Maybe someone else has a idea?

Some thoughts by me:
* I still like the "total" line. I think it is interesting to see if some episode performs better than others.
* You are right, i should avoid "subscribers" for this chart. It's a other visualization of the table of downloads

eteubert commented:
Looks better now :) I usually look at the "1w" numbers bot looking at your graph at least, the trend is the same as for 1d/2d.

I think I would like the "total" data more as bars in the background -- because rendering them as a line suggests they show a trend, which they are not.

poschi3 commented:
![image](https://user-images.githubusercontent.com/825911/94372634-99af6000-00ff-11eb-896f-9a292addc43e.png)

Now we have a bar chart and I removed 2d line. I'm sorry that the colors are different to the other charts, but it was not possible to get acceptable contrast values between these colors... The problem is, that we have two colors above (line on bar). If you want, you can pic a good bar chart color and [find better line chart values with this website](http://colorsafe.co/). But I think it's ok.

eteubert commented:
Nice! Will have a close look soon but I think this can be merged and released like that 👍🏻
